src/main/java/frc/robot/subsystems/manipulator/ManipulatorSubsystem.java

```diff
@@ -213,6 +213,14 @@ public Command getPodiumShootCommand() {
     return getShootCommand(ArmPosition.PODIUM_SHOOT, ShooterVelocity.PODIUM_SHOOT);
   }
 
+  public Command getFarPodiumShootCommand() {
```

Comment:
What is the difference between a far and non-far shot?

Reply:
well the far shot is farther :)   It's like the podium shot but a little bit farther away.    But then when I went to tune, it was very close to podium anyway.